src/views/List.jsx

```diff
@@ -1,20 +1,31 @@
 import { ListItem } from '../components';
 
 export function List({ data, listToken }) {
-	return (
-		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<p>Your token is: {listToken}</p>
-			{data.length === 0 && (
-				<p>Your shopping list is empty. Click on "Add item" to begin!</p>
-			)}
-			<ul>
-				{data.map((item) => (
-					<ListItem key={item.id} name={item.name} />
-				))}
-			</ul>
-		</>
-	);
+
+    console.log(data);
+    // OPTION B - GOES WITH OPTION B IN Firebase.js and OPTION B IN Home.jsx
+	// FILTER OUT THE DATA THAT CONTAINS AN OBJECT WITH NO NAME PROPERTY
+    const realData = data.filter((item) => item.name);
+    
+
+    return (
+        <>
+            <p>
+                Hello from the <code>/list</code> page!
+            </p>
```

Comment:
Thank you so much Caitlin! Should we also remove OPTION A (code and comments) re: `createNewList` function? We weren't sure if OPTION B version would pass the "good code" test since it relies on adding an empty object (which we then filter out in the `List`). It made sense to write the function that way because it reflects Firestore's behavior when operated manually (when adding a collection manually it can't be saved unless it contains an item) but we wanted to have a mentor' approval first. 😊

Reply:
Regarding the sanitizing code, since the other PR was already merged, it should not matter if we include the regex on our end as well, no? I'm assuming it will only create an extra conflict. (Speaking of conflicts, I think I finally got my wish. THREE conflicts! Many thanks to you capricious gods of coding!  😂)

Reply:
@CorinaMurg - 

1) I thought you all might be leaving those 'Option A' tips in the comments in case the issues Aslan described became an issue. If that's the reason, I don't mind leaving the comments there to make things easier for your future selves. We can remove them if we find it's not an issue.

As for whether this passes the 'good code' test, I don't personally have a problem with it. This approach was mentioned as an acceptable one in the ticket. If it breaks something, then we can revisit 😅 

2) The other team added sanitizing for the filter input, but we could also add it for the token input. If you wanted to add it to the token input, there are three options I see:

	a) Add the regex to the token input, and possibly DRY up the code in a future PR by pulling the logic out to a helper function.
	b) Pull the other pair's code into your branch to address merge conflicts. Once they've been fixed, pull the regex logic out to a helper function and use it for the token input too.
	c) Merge your PR as-is, and create a follow-up PR to create the helper function and add it to the token input.

Since you already plan on working through merge conflicts, maybe Option B makes sense here. You can always start working through the merge conflicts, and if you're not able to get to the sanitizing you can leave it for the future!

Will leave this up to you and @rdpfeifle ! Good luck with the merge conflicts 🎉 😆  and ping me on Slack if you run into any issues!!

Reply:
All done! Thank you for the thorough explanations, they were super helpful!